packages/docsearch-react/src/DocSearchModal.tsx

```diff
@@ -27,6 +27,13 @@ export interface DocSearchModalProps extends DocSearchProps {
   onClose?(): void;
 }
 
+const setRealHeight = () => {
+  // We update the value of the CSS variables
+  const vh = window.innerHeight * 0.01;
+  const modalElement = document.querySelector<HTMLElement>('.DocSearch-Modal')!;
```

Comment:
We should rely on CSS classes as least as possible in this piece of code. We can [add a React ref](https://github.com/francoischalifour/autocomplete.js/blob/d6c46747a4c5f0876dd47fe5dea157bd00c44b0a/packages/docsearch-react/src/DocSearchModal.tsx#L55) to get the HTML element to attach the CSS property to.

(Note that you'll need to wrap the function code with `if (modalRef.current)` to make sure that the node exists before manipulating its style, instead of forcing the type with `!`.)

Reply:
Thanks for the feedback. Definitely agreed. Update and ready for review :)